test/utils.js

```diff
@@ -0,0 +1,8 @@
+const assert = require('assert');
+
+const expectError = (promise) =>
+  promise.then(() => assert.fail("Shouldn't have gotten here."), (e) => e);
```

Comment:
You might also consider using the `chai-as-promised` plugin:

https://www.chaijs.com/plugins/chai-as-promised/

This encapsulates the pattern of "failing when an exception didn't occur".

Reply:
done